pkg/jx/cmd/step_split_monorepo.go

```diff
@@ -200,6 +204,20 @@ func (o *StepSplitMonorepoOptions) Run() error {
 					o.Git().Add(outPath, "src", "charts", "*")
 
 					message := "generated by: jx step split monorepo"
+					if reponame != "" {
+                        opt := &github.CommitsListOptions{
+                                Path:   name,
+                                ListOptions: github.ListOptions {
+                                    Page: 1,
+                                    PerPage: 1,
+                                },
+                            }
+						commits, err := gitProvider.ListCommits(organisation, reponame, opt)
+						if err != nil {
+							return err
+						}
+						message = *commits[0].Commit.Message + " - " + *commits[0].SHA
```

Comment:
You should check if the `commits` is not nil and the length is at least 1 to avoid a panic here.